test/Microsoft.TestPlatform.AcceptanceTests/BlameDataCollectorTests.cs

```diff
@@ -158,9 +171,9 @@ public void CrashDumpOnExit(RunnerInfo runnerInfo)
         }
 
         [TestMethod]
-        [NetFrameworkRunner("net452;net472;netcoreapp3.1;net5.0")]
+        [NetCoreRunner("net452;net472;netcoreapp3.1;net5.0")]
```

Comment:
What is the reason of changing attributes order?

Reply:
.net framework does not run on Linux, unless we want to test it on mono